### ISSUE TITLE
Nvidia mucks with libEGL, preventing a clean build.

### ISSUE THREAD

**Issue by zv** [das-labor/panopticon#149]
TL;DR -- Nvidia fucks up linker runtime bindings, should we fix or make substantial change to user system?

Before I wrote a PR, I thought I'd get your input on this -- it's most certainly not a problem with Panopticon, but because @flanfly is maintaining a branch of `qmlrs`, I didn't just report it upstream.

Although it is not required for the operation of Panopticon, the build process of `qmlrs` references a file in `/usr/lib64/libEGL.so`. The problem is that systems using the `nvidia-installed` have introduced a a new shared object while removing the file pointed to by the existing symbolic link (`/usr/lib64/libEGL.so` ), effectively leaving the library empty and causing the build process to fail.

There are at least three ways to tackle this in my eyes, and although I don't know that this problem is specific to Fedora systems, I would be happy to specialize `build.rs` for Fedora systems in the interim if you don't want this logic creeping into `qmlrs` proper.

Anyway, here's what could be done:

1. Prompt the user
2. Modify the Cmake script to no longer require `libEGL`, this could be as simple as removing the "`stat`" call (this builds and runs just fine) or actually changing the dependencies and dependency checking machinery.
3. Configure run-time linker bindings, which itself could be done in a number of ways (`ldconfig`, writing a `ld.conf` entry, and so on)

**Comment by flanfly** [comment]
Hey @zv, modifying the `build.rs` of qmlrs is ok. The fork is a mess anyway and I'm only maintaining it until the upstream version supports singleton types and signals.
